e2e/terraform/modules/gcp/main.tf

```diff
@@ -7,8 +21,8 @@ resource "random_string" "vm-name" {
 }
 
 locals {
-  vm-name = "e2e-vm-${random_string.vm-name.result}"
-  nephioyaml = "/home/${var.home_user}/nephio.yaml"
+  vm-name    = "e2e-vm-${random_string.vm-name.result}"
+  nephioyaml = "/home/${var.ansible_user}/nephio.yaml"
```

Comment:
are you assuming that the ansible_user and the home_user would always be the same?If yes, makes sense to combine

Reply:
That's right, there is no sense to have them separately